x/stakeibc/keeper/msg_server_redeem_stake.go

```diff
@@ -89,16 +82,18 @@ func (k msgServer) RedeemStake(goCtx context.Context, msg *types.MsgRedeemStake)
 		k.Logger(ctx).Info(fmt.Sprintf("UserRedemptionRecord found for %s", redemptionId))
 		// Add the unbonded amount to the UserRedemptionRecord
 		// The record is set below
+		userRedemptionRecord.StTokenAmount = userRedemptionRecord.StTokenAmount.Add(msg.Amount)
 		userRedemptionRecord.Amount = userRedemptionRecord.Amount.Add(nativeAmount)
 	} else {
 		// First time a user is redeeming this epoch
 		userRedemptionRecord = recordstypes.UserRedemptionRecord{
-			Id:          redemptionId,
-			Receiver:    msg.Receiver,
-			Amount:      nativeAmount,
-			Denom:       hostZone.HostDenom,
-			HostZoneId:  hostZone.ChainId,
-			EpochNumber: epochTracker.EpochNumber,
+			Id:            redemptionId,
+			Receiver:      msg.Receiver,
+			Amount:        nativeAmount,
```

Comment:
not important for this PR - at some point we should rename all "amount"s to either native amount or stToken amount 

in this file, Amount is native amount (here) and stToken amount (above on MsgRedeemStake)

very confusing

Reply:
I made the same comment on Vishals' original PR so he opened one to fix this, maybe we can do the same here (shuld be quick) https://github.com/Stride-Labs/stride/pull/1051

Reply:
We should keep in mind that we'll need to update the FE as well if we change the names on these protos

Reply:
agreed! was planning on doing this in a separate PR along with a couple other renaming's mentioned in todos